go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -282,13 +282,19 @@ func (oc *DefaultNetworkController) deleteEgressFirewall(egressFirewallObj *egre
 	return nil
 }
 
-func (oc *DefaultNetworkController) updateEgressFirewallStatusWithRetry(egressfirewall *egressfirewallapi.EgressFirewall) error {
+func (oc *DefaultNetworkController) updateEgressFirewallStatusWithRetry(egressFirewall *egressfirewallapi.EgressFirewall) error {
 	retryErr := retry.RetryOnConflict(retry.DefaultRetry, func() error {
-		return oc.kube.UpdateEgressFirewall(egressfirewall)
+		ef, err := oc.watchFactory.GetEgressFirewall(egressFirewall.Namespace, egressFirewall.Name)
+		if err != nil {
+			return err
+		}
+		c := ef.DeepCopy()
+		c.Status.Status = egressFirewall.Status.Status
+		return oc.kube.UpdateEgressFirewall(egressFirewall)
```

Comment:
the test can be just create kind with ovn-ic and check the logs, because every ovnkube-controller will try to update the same egress firewall object. With the current version this probably will fail

Reply:
oops my bad